flutter_local_notifications_windows/lib/src/ffi/bindings.dart

```diff
@@ -0,0 +1,367 @@
+
+  /// Gets all notifications that have already been shown but are still in the Action center.
+  ///
+  /// Only applications with "package identity" (ie, installed with an MSIX installer), can use this.
+  /// When your app does not have identity, such as in debug mode, this will return an empty array.
+  ffi.Pointer<NativeNotificationDetails> getActiveNotifications(
+    ffi.Pointer<NativePlugin> plugin,
+    ffi.Pointer<ffi.Int> size,
+  ) {
+    return _getActiveNotifications(
+      plugin,
+      size,
+    );
+  }
+
+  late final _getActiveNotificationsPtr = _lookup<
+      ffi.NativeFunction<
+          ffi.Pointer<NativeNotificationDetails> Function(
+              ffi.Pointer<NativePlugin>,
+              ffi.Pointer<ffi.Int>)>>('getActiveNotifications');
+  late final _getActiveNotifications = _getActiveNotificationsPtr.asFunction<
+      ffi.Pointer<NativeNotificationDetails> Function(
+          ffi.Pointer<NativePlugin>, ffi.Pointer<ffi.Int>)>();
+
+  /// Gets all notifications that have been scheduled but not yet shown.
+  ffi.Pointer<NativeNotificationDetails> getPendingNotifications(
+    ffi.Pointer<NativePlugin> plugin,
+    ffi.Pointer<ffi.Int> size,
+  ) {
+    return _getPendingNotifications(
+      plugin,
+      size,
+    );
+  }
+
+  late final _getPendingNotificationsPtr = _lookup<
+      ffi.NativeFunction<
+          ffi.Pointer<NativeNotificationDetails> Function(
+              ffi.Pointer<NativePlugin>,
+              ffi.Pointer<ffi.Int>)>>('getPendingNotifications');
+  late final _getPendingNotifications = _getPendingNotificationsPtr.asFunction<
+      ffi.Pointer<NativeNotificationDetails> Function(
+          ffi.Pointer<NativePlugin>, ffi.Pointer<ffi.Int>)>();
+
+  /// Releases the memory associated with a [NativeNotificationDetails] array.
+  void freeDetailsArray(
+    ffi.Pointer<NativeNotificationDetails> ptr,
+  ) {
+    return _freeDetailsArray(
+      ptr,
+    );
+  }
+
+  late final _freeDetailsArrayPtr = _lookup<
+      ffi.NativeFunction<
+          ffi.Void Function(
+              ffi.Pointer<NativeNotificationDetails>)>>('freeDetailsArray');
+  late final _freeDetailsArray = _freeDetailsArrayPtr
+      .asFunction<void Function(ffi.Pointer<NativeNotificationDetails>)>();
+
+  /// Releases the memory associated with a [NativeLaunchDetails].
+  void freeLaunchDetails(
+    NativeLaunchDetails details,
+  ) {
+    return _freeLaunchDetails(
+      details,
+    );
+  }
+
+  late final _freeLaunchDetailsPtr =
+      _lookup<ffi.NativeFunction<ffi.Void Function(NativeLaunchDetails)>>(
+          'freeLaunchDetails');
+  late final _freeLaunchDetails =
+      _freeLaunchDetailsPtr.asFunction<void Function(NativeLaunchDetails)>();
+
+  /// EXPERIMENTAL: Enables multithreading for this application.
+  ///
+  /// NOTE: This is only to make tests more stable and is not intended to be used in applications.
+  void enableMultithreading() {
+    return _enableMultithreading();
```

Comment:
Given the docs on this, I'm wondering if an assertion should also to also help act as another warning that developers. Some devs will not check API docs and would more likely run an app in debug mode that  if an assertion is done here, they can be given a more explicit warning/error

Reply:
This file is auto-generated and is not part of the public API. The public section is `lib/src/plugin/base.dart`, which is marked with `@visibleForTesting`. That will give a warning in the IDE/`dart analyze` if someone tries to use this.